.gitignore

```diff
@@ -40,6 +40,7 @@ balenarc.yml
 build/
 build-bin/
 build-zip/
+dist/
```

Comment:
`oclif` places the native installers under the `dist` folder. I've changed the build scripts so that the standalone zip packages are placed in the dist folder as well. The `build-zip` folder is no longer used, but keeping it in `.gitignore` for now as people will still have that folder around as they update the CLI.